### PR TITLE
Add reconciliation window fields to GatewayAccount

### PR DESCRIPTION
**Pull request by edmandiesamonte**

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -571,7 +572,7 @@ protected function getFakeValue($attribute, $class)
             case 'taxCategoryId':
                 return $faker->randomElement(Entities\Product::allowedTaxCategories());
             case 'accountingCode':
-                return (string) $faker->numberBetween(1000, 10000);
+                return (string)$faker->numberBetween(1000, 10000);
```
Comment (ashkarpetin) [inline tests/TestCase.php:575]:
```suggestion
                return (string) $faker->numberBetween(1000, 10000);
```

please check after using autoformatting

File: tests/Entities/GatewayAccountTest.php [hunk 2]
```diff
@@ -63,6 +67,8 @@ private static function getDefaultData(): array
                 'path' => '/transaction/websiteId',
                 'value' => 'website-1',
             ],
+            'reconciliationWindowEnabled' => true,
+            'reconciliationWindowTtl' => 3600
```
Comment (ashkarpetin) [inline tests/Entities/GatewayAccountTest.php:71]:
```suggestion
            'reconciliationWindowTtl' => 3600,
```

File: tests/TestCase.php [hunk 3]
```diff
@@ -331,6 +332,7 @@ protected function getFakeValue($attribute, $class)
                             sprintf('Cannot generate fake value for "%s :: %s"', $class, $attribute)
                         );
                 }
+                // no break
```
Comment (ashkarpetin) [inline tests/TestCase.php:335]: So what is the correct indent 😄  ?

Comment (edmandiesamonte) [reply tests/TestCase.php:335]: Weird that cs-fix don't detect the others. Hangon. I'll just revert those chunks to not clutter the PR. 😅

File: tests/TestCase.php [hunk 4]
```diff
@@ -355,7 +357,7 @@ protected function getFakeValue($attribute, $class)
                             sprintf('Cannot generate fake value for "%s :: %s"', $class, $attribute)
                         );
                 }
-                // no break
+            // no break
```
Comment (ashkarpetin) [inline tests/TestCase.php:360]: Previous indent was correct

**Review by ashkarpetin — APPROVED** [review]

**Review by VladimirBerdnik — APPROVED** [review]

**Pull request merged**